command/root.go

```diff
@@ -351,3 +352,37 @@ func determineEditor(cmd *cobra.Command) (string, error) {
 
 	return editorCommand, nil
 }
+
+func ExpandAlias(args []string) ([]string, error) {
+	// TODO i don't love that context is being init'd twice. My reading of the code is that it is
+	// idempotent and okay but I don't know if that's ok to depend on.
+	if len(args) == 1 {
```

Comment:
I think we should use `len(args) < 2` here to cover the empty `args` scenario. I don't know if it's pratically possible for `os.Args` to be empty, but let's not risk a panic if we ever find that out.

Reply:
`os.Args` being empty is an unsettling thought. :+1: